test/functional/lib/ipfs-companion.test.js

```diff
@@ -23,6 +23,8 @@ describe('lib/ipfs-companion.js', function () {
       global.localStorage = global.localStorage || {}
       global.URL = global.URL || URL
       global.screen = { width: 1024, height: 720 }
+      global.addEventListener = () => { }
+      global.location = { hostname: 'test' }
```

Comment:
Do we need to add addEventListener in webworkwer context? If not, why do we need to add here? Or, vice-versa?